src/diff/index.js

```diff
@@ -43,6 +43,9 @@ export function diff(parentDom, newVNode, oldVNode, context, isSvg, excessDomChi
 
 	try {
 		outer: if (oldVNode.type===Fragment || newType===Fragment) {
+			// Passing the ancestorComponent here is needed for catchErrorInComponent to properly traverse upwards through fragments
+			// to find a parent Suspense
+			c = { _ancestorComponent: ancestorComponent };
```

Comment:
Can you please double check that this won't break anything?

Reply:
It is not only about error boundaries. The `ancestorComponent` is also used in `diffChildren`, `diff` and `unmount`. It is set to the diffed component as `_ ancestorComponent` prop. I need to look a bit deeper into this as I don't fully understand the possible side-effects of this change as of now

Reply:
Passing `ancestorComponent` directly now. Saves 5 bytes